jwst/ramp_fitting/ramp_fit.py

```diff
@@ -205,15 +212,20 @@ def ols_ramp_fit_multi(input_model, buffsize, save_opt, readnoise_2d, gain_2d,
         actual_segments, actual_CRs = \
         ols_ramp_fit(input_model.data, input_model.err, input_model.groupdq, input_model.pixeldq,
                      buffsize, save_opt, readnoise_2d, gain_2d, weighting,
-                          input_model.meta.instrument.name, input_model.meta.exposure.frame_time,
-                          input_model.meta.exposure.ngroups, input_model.meta.exposure.group_time,
-                          input_model.meta.exposure.groupgap, input_model.meta.exposure.nframes,
-                          input_model.meta.exposure.drop_frames1, int_times)
+                     input_model.meta.instrument.name, input_model.meta.exposure.frame_time,
+                     input_model.meta.exposure.ngroups, input_model.meta.exposure.group_time,
+                     input_model.meta.exposure.groupgap, input_model.meta.exposure.nframes,
+                     input_model.meta.exposure.drop_frames1, int_times)
+
+        # Populate the rateints output model
         if number_of_integrations > 1:
             int_model.data = int_data
             int_model.dq = int_dq
             int_model.var_poisson = int_var_poisson
             int_model.var_rnoise = int_var_rnoise
+            int_model.err = int_err
```

Comment:
Reviewers: this is the only real code change. All others are purely cosmetic for readability.